src/fixed-footer-layout.tsx

```diff
@@ -106,17 +105,17 @@ const FixedFooterLayout: React.FC<Props> = ({
         };
     }, [hasContentEnoughVSpace, platformOverrides]);
 
-    const isContentWithScroll = isClientSide() && hasScroll(getScrollableParentElement(containerRef.current));
-
-    const isFixedFooter = hasContentEnoughVSpace || !isContentWithScroll;
+    const isFixedFooter = hasContentEnoughVSpace;
```

Comment:
After debugging in different devices (mobile and web) and testing several scenarios with @brtbrt, we concluded that it's useless to check for the condition `!isContentWithScroll`. This was old logic, and it makes no sense to use it in the current code.

Reply:
specifically, it come from the old JSS styling 👍